doc/tutorials/core/basic_geometric_drawing/basic_geometric_drawing.markdown

```diff
@@ -218,22 +111,17 @@ Explanation
         -   The color of the polygon is defined by **Scalar( 255, 255, 255)**, which is the BGR
             value for *white*
     -   *rectangle*
-        @code{.cpp}
-        rectangle( rook_image,
-                   Point( 0, 7*w/8.0 ),
-                   Point( w, w),
-                   Scalar( 0, 255, 255 ),
-                   -1, 8 );
-        @endcode
+        @snippet cpp/tutorial_code/core/Matrix/Drawing_1.cpp rectangle
+
         Finally we have the @ref cv::rectangle function (we did not create a special function for
         this guy). We note that:
 
         -   The rectangle will be drawn on **rook_image**
-        -   Two opposite vertices of the rectangle are defined by *\* Point( 0, 7*w/8.0 )*\*
+        -   Two opposite vertices of the rectangle are defined by *\* Point( 0, 7*w/8 )*\*
```

Comment:
Same remark, Point has by default a double precision, and the .0 division ensures that.